src/14-curated-discovery/01-artworks.ts

```diff
@@ -269,7 +274,7 @@ async function insertArtworks(
             tags: artwork.tags,
             additionalInformation: artwork.additional_information,
             imageUrl: artwork.image_url,
-            imageDescription: null, // TODO: add computer vision step?
+            imageDescription,
```

Comment:
The image descriptions are included in the artwork creation process here, but this presumes that `src/14-curated-discovery/04-computer-vision.ts` has been run first, in order to generate the image descriptions and write them to the filesystem.